lib-compose/src/main/java/com/what3words/components/compose/maps/state/camera/W3WMapboxCameraState.kt

```diff
@@ -12,12 +13,16 @@ class W3WMapboxCameraState(override val cameraState: MapViewportState) :
     W3WCameraState<MapViewportState> {
 
     companion object {
-        const val MY_LOCATION_ZOOM = 19.0
+        const val MY_LOCATION_ZOOM = 20.0
     }
 
     override var gridBound: W3WRectangle? = null
 
-    override fun orientCamera() {
+    //TODO: This is work around for the function cameraForCoordinates not support in compose
+    var cameraForCoordinates:MutableList<Point>? = mutableListOf()
+
+    @UiThread
+    override suspend fun orientCamera() {
```

Comment:
The @UiThread annotation in Android is a contract or indicator, not a guarantee and it does not enforce it at runtime or switch to the main thread automatically. To ensure thread safety and compliance with coroutine rules, please modify the function to explicitly run on the Dispatchers.Main dispatcher